### ISSUE TITLE
Segmentation fault when running libbi for the first time

### ISSUE THREAD

**Issue by tcquinn** [lawmurray/LibBi#43]
I installed LibBi on my Linux system following the instructions in INSTALL_LINUX.md and INSTALL.md. Installation appeared to be successful, with no critical errors reported, and all tests were passed at the end of the build.

To test the installation, I downloaded the Windkessel example. `libbi sample @config.conf @prior.conf` seemed to run just fine (no errors reported) but `libbi sample @config.conf @posterior.conf` generates:
```
Warning: Cholesky factorisation exception sampling trajectory
Segmentation fault
```
A few details of my system:
Operating system: Linux Mint 18 Cinnamon 32-bit
Linux kernel: 4.4.0-98-generic
Processor: Intel Core 2 Duo CPU P8700 @ 2.53GHz x 2
Memory: 4 Gb

**Comment by sbfnk** [comment]
Thanks for reporting this issue, @tcquinn. Could you check if installing LibBi from the `develop` branch fixes it?

**Comment by tcquinn** [comment]
Thanks for following up on this. I cloned and installed from the `develop` branch on an Ubuntu Server 16.04 LTS instance and it seemed to work. I ran the Windkessel example and I got one warning (`Warning: Cholesky factorisation exception sampling trajectory`), but it seemed to finish successfully and generate results. I haven't done any sanity checks on the results or tried any other examples, but I've got it running, so I'm happy to close this issue unless you'd like me to poke around in logs or anything to figure out why the `develop` branch succeeded where the `master` branch failed. Thank you!
